imports/plugins/included/default-theme/client/styles/navbar.less

```diff
@@ -261,3 +247,57 @@ html:not(.rtl) .rui.navbar .languages .dropdown-menu {
 .rui.navbar .search:hover button {
   outline: none;
 }
+
+.rui.navbar .accounts:hover {
+  background-color: @navbar-default-toggle-icon-bar-bg;
+  border-color: @navbar-default-toggle-border-color;
+}
+
+.accounts-a-tag {
+  display: flex !important;
+  align-items: center;
+  color: @gray !important;
+  text-decoration: none;
+  padding: 5px 0 !important;
+  min-height: 44px !important;
+  min-width: 220px;
+}
+
+.accounts-a-tag:hover,
+.accounts-a-tag:focus {
+  color: @brand-vivid-color !important;
+  text-decoration: none;
+  background-color: transparent !important;
+}
+
+.accounts-li-tag {
+  cursor: pointer;
+  white-space: nowrap;
+  display: block;
+  border-bottom: 1px solid @black10;
+}
+
+.accounts-li-tag:last-child {
+  border-bottom: none;
+}
+
+.accounts-btn-tag {
+  padding: 6px 12px !important;
+  color: #fff !important;
+  background-color: @btn-primary-bg !important;
```

Comment:
Right, I understand what `important` does. Is there no way to make the selectors more specific? What happens when we want to override those styles?